bluebutton-data-model-rif/src/main/resources/db/migration/V10__Create_Dataload_history_table.sql

```diff
@@ -0,0 +1,18 @@
+/*
+ * Creates the dataload history table required by the application. This
+ * schema is derived from the JPA metadata, via the 
+ * gov.hhs.cms.bluebutton.data.pipeline.rif.schema.HibernateSchemaPrinter 
+ * utility.
+ */
+
+${logic.tablespaces-escape} SET default_tablespace = fhirdb_ts2;
```

Comment:
Is this default tablespace declaration doing anything here? (Can't remember why some of the others have it.)